src/WebJobs.Script/Binding/ServiceBusScriptBindingProvider.cs

```diff
@@ -140,25 +140,27 @@ public override Collection<Attribute> GetAttributes()
 
                 if (Context.IsTrigger)
                 {
-                    attributes.Add(new EventHubTriggerAttribute(eventHubName));
-
-                    string eventProcessorHostName = Guid.NewGuid().ToString();
-                    string storageConnectionString = _storageConnectionString;
-
                     string consumerGroup = Context.GetMetadataValue<string>("consumerGroup");
                     if (consumerGroup == null)
                     {
                         consumerGroup = Microsoft.ServiceBus.Messaging.EventHubConsumerGroup.DefaultGroupName;
                     }
 
+                    string keyName = eventHubName + "-" + consumerGroup;
+
+                    attributes.Add(new EventHubTriggerAttribute(keyName));
```

Comment:
As discussed, rather than this tactical fix (which isn't really correct), we should do the full fix for this.